### PR TITLE
Rename tenant cluster to workload cluster

### PR DESCRIPTION
**Pull request by marians**
Towards https://github.com/giantswarm/giantswarm/issues/14961

For the generated content in the Changes and Releases section, there is https://github.com/giantswarm/docs/pull/740

### CODE REVIEW

File: scripts/aggregate-changelogs/config.yaml [hunk 1]
```diff
@@ -92,11 +92,11 @@ categories:
     color: "#bfffb8"
   - name: Rest API
     color: "#919ea1"
-  - name: Tenant Cluster Releases for AWS
+  - name: Workload Cluster Releases for AWS
```
Comment (JosephSalisbury) [inline scripts/aggregate-changelogs/config.yaml:95]: is capitalisation funny here? would it be `Workload cluster releases for AWS`?

File: scripts/aggregate-changelogs/script.py [hunk 2]
```diff
@@ -221,16 +221,18 @@ def generate_release_file(repo_shortname, repo_config, release):
...
+    aliases = None
     
     if repo_shortname == RELEASES_REPO:
         provider_label = release['provider'].upper()
         if provider_label == 'AZURE':
             provider_label = 'Azure'
-        categories = [f'Tenant Cluster Releases for {provider_label}']
-        title = f'Tenant Cluster Release v{version} for {provider_label}'
+        categories = [f'Workload Cluster Releases for {provider_label}']
+        title = f'Workload Cluster Release v{version} for {provider_label}'
```
Comment (JosephSalisbury) [inline scripts/aggregate-changelogs/script.py:231]: and sort of continuing here

File: scripts/aggregate-changelogs/config.yaml (hunk 1 above)
Comment (marians) [reply scripts/aggregate-changelogs/config.yaml:95]: Maybe, however that hasn't really changed in this PR and I don't want to include changes of many category names in this PR.

image

Comment (JosephSalisbury) [reply scripts/aggregate-changelogs/config.yaml:95]: Sure, works for me

**Review by JosephSalisbury — APPROVED** [review]

**Pull request merged**